randovania/games/am2r/json_data/Industrial Complex.txt

```diff
@@ -745,7 +745,23 @@ Extra - minimap_data: [{'x': 66, 'y': 26}, {'x': 67, 'y': 26}, {'x': 68, 'y': 26
   * Layers: default
   * Open Passage to Upper Factory Autoad Fork/Dock to Shirk Prisons
   > Dock to Treadmill Room
-      Missiles ≥ 2 and Speed Booster
+      All of the following:
+          Speed Booster
+          Any of the following:
+              # Speedboost on far right
+              Missiles ≥ 2 and Morph Ball
```

Comment:
Do we also need a supers case? Do we have a template of `destroy missile blocks` we can use? Not sure how we'd do that for ammo x2 or x4 though

Reply:
A "shoot any missile" template wouldnt help here, as then some weird cases can happen due to that.
and yes, we want supers for those too